src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -1,17 +1,35 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
-  <PropertyGroup>
-    <PackageId>Monitor.Tests</PackageId>
-    <Description>Monitor.Tests Class library</Description>
-    <PackageId>Monitor.Tests</PackageId>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
-  </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Management.Monitor\Microsoft.Azure.Management.Monitor.csproj" />
-    <None Update="SessionRecords\**\*.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
-  </ItemGroup>
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+	<PropertyGroup>
+		<PackageId>Monitor.Tests</PackageId>
+		<Description>Monitor.Tests Class library</Description>
+		<PackageId>Monitor.Tests</PackageId>
+		<VersionPrefix>1.0.0-preview</VersionPrefix>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452</TargetFrameworks>
```

Comment:
My version is 15.1 at the moment. I'll try with 15.2.
The tests are discovered and run fine for .net452. That's how I re-recorded them. 
It is only netcoreapp1.1 that does not detect any tests locally. However, in your side, during build, the system apparently discovers them and tries to run the tests for netcoreapp1.1 but fails to find the dll, it looks in a folder where, I assume, the dll is not installed.